components/list/list.js

```diff
@@ -224,13 +225,14 @@ class List extends PageableMixin(SelectionMixin(LitElement)) {
 			e.stopPropagation();
 		}
 		const items = this.getItems();
+		let aChildHasColor = false;
 		let aChildHasToggleEnabled = false;
 		for (const item of items) {
```

Comment:
I wish we didn't have to do this looping. There's some discussion of adding a CSS API to detect when slots contains things.

Reply:
This handles updating the siblings whenever items are added or removed from the slot. Do we have something for handling the scenario where the list loads, and then later a color is added or removed from an item? For example in response to something the user has done.